manifests/init.pp

```diff
@@ -84,18 +77,18 @@
     fedora: {
       # Fedora distros < 22 are EOL as of 2015-12-01
       # https://fedoraproject.org/wiki/End_of_life
-      if $::operatingsystemmajrelease < '22' {
-        fail("Unsupported OS: ${::operatingsystem} ${::operatingsystemmajrelease}")
+      if $::lsbmajdistrelease < '22' {
+        fail("Unsupported OS: ${::lsbdistid} ${::lsbmajdistrelease}")
       }
     }
     ubuntu: {
-      if $::operatingsystemmajrelease != '14.04' {
-        # Only tested on 14.04
-        fail("Unsupported OS: ${::operatingsystem} ${::operatingsystemmajrelease}")
+      if $::lsbdistrelease < '16.04' {
+        # Only tested on 16.04
+        fail("Unsupported OS: ${::lsbdistid} ${::lsbdistrelease}")
       }
     }
     default: {
-      fail("Unsupported OS: ${::operatingsystem}")
+      fail("Unsupported OS: ${::lsbdistid}")
```

Comment:
After running `$BEAKER_destroy=no bundle exec rake cent_6test_vm`, I tried:

```
~/puppet-opendaylight/.vagrant/beaker_vagrant_files/centos-7.yml$ vagrant ssh
[vagrant@centos-7 ~]$ facter lsbdistid

[vagrant@centos-7 ~]$ facter operatingsystem
CentOS
```
The `lsbdist*` facts are missing by default in CentOS. We need to install `redhat-lsb-core` package for these vars to be available. After adding,
```
docker_image_commands:
      - 'yum install -y redhat-lsb-core'
```
to `nodesets/centos-7-docker.yml`, all tests are successful for the CentOS docker containers. Need to figure out how to install the package for CentOS vagrant boxes.

Is it better to have only `operatingsystem*` vars for centos/redhat and both `lsbdist*` and `operatingsystem*` vars for ubuntu? In this case, we might not need to install `redhat-lsb-core` package, but there will be duplication and probably difficulty in maintaining the code. What do you think?